fs/fs_zenfs.cc

```diff
@@ -287,6 +287,23 @@ IOStatus ZenFS::WriteEndRecord(ZenMetaLog* meta_log) {
   return meta_log->AddRecord(endRecord);
 }
 
+/* Assumes the files_mtx_ is held */
+IOStatus ZenFS::RollMetaZone() {
+#ifdef WITH_ZENFS_ASYNC_METAZONE_ROLLOVER
+  return RollMetaZoneAsync();
+#else
+  return RollMetaZoneLocked();
+#endif // WITH_ZENFS_ASYNC_METAZONE_ROLLOVER
+}
+
+#ifdef WITH_ZENFS_ASYNC_METAZONE_ROLLOVER
```

Comment:
@weilewei This macro is not needed for the function definition, you already control the access in the entry function.